sdk/identity/identity/interactive-browser-credential.md

```diff
@@ -1,18 +1,60 @@
 # Interactive Browser Credential
 
-The `InteractiveBrowserCredential` uses [Auth Code Flow][AuthCodeFlow], which uses [Proof Key for Code Exchange (PKCE)](https://tools.ietf.org/html/rfc7636), uses [MSAL v2.x](https://github.com/AzureAD/microsoft-authentication-library-for-js), and is enabled by default. It also allows switching back to the [Implicit Grant Flow][ImplicitGrantFlow] by passing the `flow` property with the value `implicit-grant` through to the constructor of the `InteractiveBrowserCredential`.
+The `InteractiveBrowserCredential` uses [Authorization Code Flow][AuthCodeFlow], which uses [Proof Key for Code Exchange (PKCE)](https://tools.ietf.org/html/rfc7636), uses [MSAL v2.x](https://github.com/AzureAD/microsoft-authentication-library-for-js), and is enabled by default. It also allows switching back to the [Implicit Grant Flow][ImplicitGrantFlow] by passing the `flow` property with the value `implicit-grant` through to the constructor of the `InteractiveBrowserCredential`, as follows:
+
+```ts
+const credential = new InteractiveBrowserCredential({
+  // Authorization Code Flow is recommended and used by default.
+  // But you can switch batch to the Implicit Grant Flow if you need to:
+  flow: "implicit-grant",
+});
+```
 
 Follow the instructions for [creating your single-page application](https://docs.microsoft.com/azure/active-directory/develop/scenario-spa-app-registration#redirect-uri-msaljs-20-with-auth-code-flow) to correctly mark your redirect URI as enabled for CORS.
 
-If you attempt to use the authorization code flow and see this error:
+When using `InteractiveBrowserCredential` on Node, you may specify a `clientId` and `tenantId`, but otherwise we try to authenticate using a public client that's available for all Azure accounts and the default tenant of your account. For Node, this credential uses a web server to fulfill the redirection. This web server tries to use the port `80` by default. A `redirectUri` can be provided to determine the proper redirection URI with the adequate port, as follows:
```

Comment:
The default values do work in Node. Here's a simple that can be used to test it:

```ts
const identity = require("@azure/identity");
const keys = require("@azure/keyvault-keys");

const credential = new identity.InteractiveBrowserCredential({ redirectUri: "http://localhost:8086/" });
const client = new keys.KeyClient("https://vault-name.vault.azure.net/", credential);

(async () => {
  await client.getKey("keyName");
})().then(console.log).catch(console.log)
```